Sources/generate-swift-syntax-builder/Templates/BuildableNodesFile.swift

```diff
@@ -0,0 +1,372 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import SwiftSyntax
+import SwiftSyntaxBuilder
+
+let buildableNodesFile = SourceFile {
+  ImportDecl(
+    leadingTrivia: .docLineComment(copyrightHeader),
+    path: "SwiftSyntax"
+  )
+
+  for node in SYNTAX_NODES where node.isBuildable {
+    let type = node.type
+    let baseType = node.baseType
+    let hasTrailingComma = node.traits.contains("WithTrailingComma")
+    let conformances = [baseType.buildableBaseName, type.expressibleAsBaseName] + (hasTrailingComma ? ["HasTrailingComma"] : [])
+
+    // Generate node struct
+    StructDecl(
+      leadingTrivia: node.documentation.isEmpty
+        ? []
+        : .docLineComment("/// \(node.documentation)") + .newline,
+      modifiers: [TokenSyntax.public],
+      identifier: type.buildableBaseName,
+      inheritanceClause: createTypeInheritanceClause(conformances: conformances)
+    ) {
+      let children = node.children
```

Comment:
Is there any specific reason why we need the `children` variable and can’t just use `node.children` below?